### PR TITLE
Optimism token list 

### PR DESCRIPTION
**Pull request by lkulier**

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+      "name": "Frax",
+      "symbol": "FRAX",
+      "decimals": 18,
+      "logoURI": "https://assets-stage.dex.guru/icons/0x2e3d870790dc77a83dd1d18184acc7439a53f475-optimism.png"
+    },
+    {
+      "chainId": 10,
+      "address": "0xB0B195aEFA3650A6908f15CdaC7D92F8a5791B0B",
+      "name": " BOB",
+      "symbol": " BOB",
```
Comment (arddluma) [inline optimism.tokenlist.json]: Please remove the space heree

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0xFdb794692724153d1488CcdBE0C56c252596735F",
+      "name": "Lido DAO Token",
+      "symbol": "LDO",
+      "decimals": 18,
+      "logoURI": "https://assets-stage.dex.guru/icons/0xfdb794692724153d1488ccdbe0c56c252596735f-optimism.png"
+    },
+    {
+      "chainId": 10,
+      "address": "0xc40F949F8a4e094D1b49a23ea9241D289B7b2819",
+      "name": "LUSD Stablecoin",
```
Comment (arddluma) [inline optimism.tokenlist.json]: Keep only LUSD

File: optimism.tokenlist.json [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0x3c8B650257cFb5f272f799F5e2b4e65093a11a05",
+      "name": "Velodrome",
+      "symbol": "VELO",
+      "decimals": 18,
+      "logoURI": "https://app.velodrome.finance/images/wordmark_white.svg"
+    },
+    {
+      "chainId": 10,
+      "address": "0xdFA46478F9e5EA86d57387849598dbFB2e964b02",
+      "name": "Mai Stablecoin",
```
Comment (arddluma) [inline optimism.tokenlist.json]: Remove Stablecoin

File: optimism.tokenlist.json [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0x9e5AAC1Ba1a2e6aEd6b32689DFcF62A509Ca96f3",
+      "name": "dForce",
+      "symbol": "DF",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/data/DF/logo.svg"
+    },
+    {
+      "chainId": 10,
+      "address": "0x8aE125E8653821E851F12A49F7765db9a9ce7384",
+      "name": "Dola USD Stablecoin",
```
Comment (arddluma) [inline optimism.tokenlist.json]: Remove Stablecoin

File: optimism.tokenlist.json [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0x0994206dfe8de6ec6920ff4d779b0d950605fb53",
+      "name": "Curve DAO Token",
+      "symbol": "CRV",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/data/CRV/logo.png"
+    },
+    {
+      "chainId": 10,
+      "address": "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1",
+      "name": "Dai stable coin",
```
Comment (arddluma) [inline optimism.tokenlist.json]: Use only DAI in the name, remove stable coin

File: optimism.tokenlist.json [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0x5029C236320b8f15eF0a657054B84d90bfBEDED3",
+      "name": "BitANT",
+      "symbol": "BitANT",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/data/BitANT/logo.png"
+    },
+    {
+      "chainId": 10,
+      "address": "0x0994206dfe8de6ec6920ff4d779b0d950605fb53",
+      "name": "Curve DAO Token",
```
Comment (arddluma) [inline optimism.tokenlist.json]: Remove `DAO Token`

File: optimism.tokenlist.json [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0x6fd9d7ad17242c41f7131d257212c54a0e816691",
+      "name": "Uniswap",
+      "symbol": "UNI",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/data/UNI/logo.png"
+    },
+    {
+      "chainId": 10,
+      "address": "0x7F5c764cBc14f9669B88837ca1490cCa17c31607",
+      "name": "USD Coin",
```
Comment (arddluma) [inline optimism.tokenlist.json]: Use `USDC` in the name

File: optimism.tokenlist.json [hunk 8]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0x217D47011b23BB961eB6D93cA9945B7501a5BB11",
+      "name": "Optimistic Thales Token",
+      "symbol": "THALES",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/data/THALES/logo.png"
+    },
+    {
+      "chainId": 10,
+      "address": "0xcB59a0A753fDB7491d5F3D794316F1adE197B21E",
+      "name": "TrueUSD",
```
Comment (arddluma) [inline optimism.tokenlist.json]: TUSD in name

File: optimism.tokenlist.json [hunk 9]
```diff
@@ -0,0 +1,364 @@
...
+      "address": "0x7F5c764cBc14f9669B88837ca1490cCa17c31607",
+      "name": "USD Coin",
+      "symbol": "USDC",
+      "decimals": 6,
+      "logoURI": "https://ethereum-optimism.github.io/data/USDC/logo.png"
+    },
+    {
+      "chainId": 10,
+      "address": "0x94b008aA00579c1307B0EF2c499aD98a8ce58e58",
+      "name": "Tether USD",
```
Comment (arddluma) [inline optimism.tokenlist.json]: USDT in name

**Review by arddluma — CHANGES_REQUESTED** [review]
As per comments

**Comment by lkulier** [comment]
Comments resolved

**Pull request merged**